TSOMessageHub/Jobs/SendXML.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Quartz;
+using System;
+using System.Data.SqlTypes;
+using System.Xml;
+using System.Xml.Serialization;
+using MassTransit;
+using TSOMessageHub.XML;
+
+namespace TSOMessageHub;
+
+public class SendXML : IJob
+{
+    private readonly ILogger<SendXML> _logger;
```

Comment:
Consider removing unused logger or finding a use for it

Reply:
Removed from class